prime-router/src/main/kotlin/azure/WorkflowEngine.kt

```diff
@@ -1,23 +1,40 @@
 package gov.cdc.prime.router.azure
 
 import gov.cdc.prime.router.CsvConverter
+import gov.cdc.prime.router.Hl7Converter
 import gov.cdc.prime.router.Metadata
 import gov.cdc.prime.router.Report
+import gov.cdc.prime.router.Translator
 import org.jooq.Configuration
 import java.io.ByteArrayInputStream
 
 /**
  * Methods to add a new report to the workflow pipeline and to handle a step in the pipeline.
+ * A new WorkflowEngine object should be created for every function call.
  *
  * @see gov.cdc.prime.router.Report
  * @see QueueAccess
  * @see DatabaseAccess.Header
  */
 class WorkflowEngine(
-    private val db: DatabaseAccess = DatabaseAccess(),
-    private val blob: BlobAccess = BlobAccess(),
-    private val queue: QueueAccess = QueueAccess(),
+    // Immutable objects can be shared between every function call
+    val metadata: Metadata = WorkflowEngine.metadata,
+    val hl7Converter: Hl7Converter = WorkflowEngine.hl7Converter,
```

Comment:
It metadata object is now loaded once, on the first call to a function. This speeds up processing considerably.

Reply:
The server process doesn't shutdown after a request.